files/default_files/forward.php

```diff
@@ -7,7 +7,7 @@
     $data['title'] = isset($_GET['title']) ? $_GET['title'] : "" ;
     $data['kwords'] = isset($_GET['kwords']) ? $_GET['kwords'] : "" ;
     $data['desc'] = isset($_GET['desc']) ? $_GET['desc'] : "" ;
-    $data['url'] = isset($_GET['url']) ? $_GET['url'] : "" ;
+    $data['url'] = isset($_GET['url']) && filter_var($_GET['url'], FILTER_VALIDATE_URL) ? $_GET['url'] : "" ;
```

Comment:
Will this work for international urls, like: `http://ätomia.com`?
If domain is not IDN encoded - it will probably fail.

Reply:
Domain will be encoded automatically by Automation Server.